README.rst

```diff
@@ -1,25 +1,23 @@
-django-menu
+Django-menu
 -----------
 
 BSD-licensed menu tools for Django, built by Ross Poulton <http://www.rossp.org>
 
-django-menu provides a basic structure for you to build multiple navigation 
-menus for your website, such as the header menubar. These menus can be easily 
-maintained by staff using the Django administration without any knowledge 
-of HTML or Django internals.
+Django-menu provides a basic structure for you to build multiple navigation menus for your websites, like the header menubar. These menus can be easily 
+maintained by the staff using the Django administration without any knowledge of HTML or Django internals.
 
-Sub-menus can also be easily built and displayed only for particular URIs.
+The Sub-menus can also be easily built and displayed only for particular URIs.
 
 Installation & Configuration:
 -----------------------------
 
-1. ``pip install django-menu``
+1. ``pip install Django-menu``
```

Comment:
Backticks (\`) are used for syntax highlighting in markdown - it generally indicates that the text in the markdown  refers to code or a typed command where capitalisation and spaces matter
In this case the command is different with an upper case `D` in `Django`
https://guides.github.com/features/mastering-markdown/